roster.md

```diff
@@ -3,6 +3,8 @@
 * Kevin Buffardi (instructor)
 * Devam Sharma (student)
 * Gopal Raigoni (student)
+* Jayesh Sathe (student)
+* Shelley Wong (student) 
```

Comment:
There's a duplicate here, where you have Shelley (incorrectly) before Padam, but also (correctly) after (line 9)

Reply:
Hi Kevin,
I have updated the names alphabetically. Please check.
Thank You.